src/environmentbase/environmentbase.py

```diff
@@ -594,7 +594,7 @@ def initialize_template(self):
         print '\nGenerating templates for {} stack\n'.format(self.globals['environment_name'])
 
         # Configure Template class with S3 settings from config
-        Template.template_bucket = self.template_args.get('s3_bucket')
+        Template.template_bucket_default = self.template_args.get('s3_bucket')
```

Comment:
@gimballock: I wonder if I should change `s3_bucket` to `s3_bucket_default` or so. What do you think?